content/docs/2.7/concepts/scaling-deployments.md

```diff
@@ -212,3 +212,15 @@ Using this method can preserve a replica and enable long-running executions.  Ho
 ### Run as jobs
 
 The other alternative to handling long running executions is by running the event driven code in Kubernetes Jobs instead of Deployments or Custom Resources.  This approach is discussed [in the next section](../scaling-jobs).
+
+### Pause autoscaling
```

Comment:
Just a nit, could you please move it a little bit up in the document? I'd put it above `## Long-running executions`, ie around line 198